examples/jitclass.py

```diff
@@ -5,36 +5,38 @@
 A simple jitclass example.
 """
 
+import typing
+
 import numpy as np
-from numba import int32, float32    # import the types
+from numba import float32
 from numba.experimental import jitclass
 
-spec = [
-    ('value', int32),               # a simple scalar field
-    ('array', float32[:]),          # an array field
-]
-
```

Comment:
I wonder if we shouldn't have both ways of doing things in this example, with type annotations and the old style spec? I do like the new style, though, so if the old one were to be phased out, I would shed no tears.

Reply:
Done.